platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -28,66 +30,102 @@
                     .toArray(String[]::new);
-            configureDemoCertWithDN(dn);
+            configureDemoCertWithDN(dn, sans);
         }
     }
 
     /**
-     * Generates new signed certificate. The input parameter is used as the certificate's common name.
+     * Generates new signed certificate. The input parameter is used as the certificate's common name
+     * and optional subject alternative names.
      * <p>
      * Postcondition is the server keystore is updated to include a private entry. The private
      * entry has the new certificate chain that connects the server to the Demo CA. The matching
      * private key is also stored in the entry.
      *
-     * @param commonName string to use as the common name in the new certificate.
+     * @param commonName      string to use as the common name in the new certificate.
+     * @param subjectAltNames names in the form {@code tag:name} (format similar to OpenSSL X509 configuration)
      * @return the string used as the common name in the new certificate
      */
-    public static String configureDemoCert(String commonName) {
+    public static String configureDemoCert(String commonName, @Nullable String[] subjectAltNames) {
         CertificateSigningRequest csr = new CertificateSigningRequest();
         csr.setCommonName(commonName);
+        if (subjectAltNames != null) {
+            csr.addSubjectAlternativeNames(subjectAltNames);
+        }
         return configureCert(commonName, csr);
     }
 
     /**
      * Generates new signed certificate. The input parameter is the full set of attributes of the
-     * distinguished name for the cert. It must include a single {@code CN} value for the common name.
+     * distinguished name for the cert and optional subject alternative names. It must include a
+     * single {@code CN} value for the common name.
      * <p>
      * Postcondition is the server keystore is updated to include a private entry. The private
      * entry has the new certificate chain that connects the server to the Demo CA. The matching
      * private key is also stored in the entry.
      *
-     * @param dn String params in the form {@code attrKey=attrVal} composing a distinguished name.
-     *           e.g. {@code configureDemoCertWithDN("cn=John Whorfin", "o=Yoyodyne", "l=San Narciso", "st=California", "c=US")}
+     * @param dn              String params in the form {@code attrKey=attrVal} composing a distinguished name.
+     *                        e.g. {@code configureDemoCertWithDN(new String[] {"cn=John Whorfin", "o=Yoyodyne", "l=San Narciso", "st=California", "c=US"), null}
+     * @param subjectAltNames names in the form {@code tag:name} (format similar to OpenSSL X509 configuration)
      * @return the string used as the common name in the new certificate
      */
-    public static String configureDemoCertWithDN(String... dn) {
+    public static String configureDemoCertWithDN(String[] dn, @Nullable String[] subjectAltNames) {
```

Comment:
I think the original name was to differentiate with the method above. Now both have optional support for SANs so not sure renaming it brings any value.